src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -77,7 +77,7 @@ private void ValidateScopeInput(ISet<string> scopesToValidate)
 
         public async Task<AuthenticationResult> RunAsync(CancellationToken cancellationToken = default)
         {
-            Stopwatch sw = Stopwatch.StartNew();
+            long currentElapsedMilliseconds = StopWatchService.CurrentElapsedMilliseconds;
```

Comment:
I know that this is exposing the internal stopwatch, but there isnt a great way to cover multiple lines like this. Are we intending to also measure logging statements/creation of telemetry clients?

Reply:
@bgavrilMS @neha-bhargava ^

Reply:
Have a look at https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/Internal/Logger/DurationLogHelper.cs#L10 for how we currently measure blocks.

Reply:
yeah, but what is interesting about this one is that the measurement takes place outside of the using statement and then  finished inside the using statement. Seems like certain lines are selected to be included in the measurement. Ill see if there is some way to refactor.